### PR TITLE
add new slider ticks topic

### LINKED ISSUE #1668: Improve the slider topic after latest functionalities
Enhance the slider topic regarding the latest functionalities added -ticks, etc.

### PR DESCRIPTION
**Pull request by Aleksandyr**
Closes #1668

### CODE REVIEW

File: en/components/slider/slider_ticks.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+```
+
+<div class="sample-container loading" style="height: 140px">
+    <iframe id="slider-secondary-ticks-mirror-iframe" data-src='{environment:demosBaseUrl}/interactions/slider-secondary-ticks-mirror' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="slider-secondary-ticks-mirror-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+In order to change the orientation, we have used [`ticksOrientation`]({environment:angularApiUrl}/classes/igxslidercomponent.html#ticksorientation) input, which we changed from **Bottom**(default) to **Mirror**. This mirrors the visualization of the **ticks** and duplicates them at the top as well.
```
Comment (hanastasov) [inline en/components/slider/slider_ticks.md]: Something simpler here ? Maybe "Ticks orientation is changed by setting the ticksOrientation input. We changed it from Bottom to Mirror to mirror the ticks and have them visualized on top as well."

File: en/components/slider/slider_ticks.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    <iframe id="slider-secondary-ticks-mirror-iframe" data-src='{environment:demosBaseUrl}/interactions/slider-secondary-ticks-mirror' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="slider-secondary-ticks-mirror-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+In order to change the orientation, we have used [`ticksOrientation`]({environment:angularApiUrl}/classes/igxslidercomponent.html#ticksorientation) input, which we changed from **Bottom**(default) to **Mirror**. This mirrors the visualization of the **ticks** and duplicates them at the top as well. 
+
+#### Top ticks with visible labels
+There is an edge case where our **thumb label** is hidden intentionally, and it is when [`ticksOrientaion`]({environment:angularApiUrl}/classes/igxslidercomponent.html#ticksorientation) is set to **Top** or **Mirror** and there are any visible **tick labels**. In this way, we prevent a bad user experience and overlap between the two labels. To gain a better view over that scenario let’s see the example below.
```
Comment (hanastasov) [inline en/components/slider/slider_ticks.md]: maybe you mean "there are NO visible tick labels" or "there arent any visible" ?

File: en/components/slider/slider_ticks.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    <igx-slider
+    [showTicks]="true"
+    [primaryTicks]="3"
+    [secondaryTicks]="3">
+        <ng-template igxSliderTickLabel let-value let-primary="isPrimary" let-idx="index">
+            {{ tickLabel(value, primary, idx) }}
+        </ng-template>
+    </igx-slider>
+```
+Setting to the template [`IgxTickLabelTemplateDirective`]({environment:angularApiUrl}/classes/igxticklabeltemplatedirective.html) says that this is the template we need to look for when we render our **tick labels**.
```
Comment (hanastasov) [inline en/components/slider/slider_ticks.md]: I think "setting to the template" will make mroe sense, I am confused what ti means here. Maybe "Setting the template"

File: en/components/slider/slider_ticks.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
+---
+title: Slider Component
+_description: Ignite UI for Angular Slider control provides a rich input for value selection and delivers consistent experience across browsers.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Slider components, Angular Slider controls
+---
+
+## Slider Ticks
+**Slider ticks**, provide a new and more attractive way for data visualization. More specifically, we could use that feature to present a particular timeframe, days of the week and more. With this new functionality we are not obligated to interact with the slider in order to see what data stays behind. It is extremely flexible. The user has a full control over the positioning of the **ticks** and **tick labels**. We can turn on/off them, as well as pick between primary, secondary or both. Users also can turn on/of primary, secondary **tick labels** or both, even they can change their rotation form **horizontal** to **vertical** (**top to bottom** (90) or **bottom to top** (-90)).
```
Comment (hanastasov) [inline en/components/slider/slider_ticks.md]: 1. Users can also
2. they can even change

**Comment by kdinev** [comment]
@Aleksandyr @yoshimis I have applied some changes. I doubt they will affect the JP translation, but could you please take a look, just in case?

**Comment by Aleksandyr** [comment]
@kdinev I agree with the changes. They all seem relevant.

File: en/components/slider/slider.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Slider Component
+title: Slider ticks | Ticks | Ignite UI for Angular | infragistics
```
Comment (zdrawku) [inline en/components/slider/slider.md]:
```suggestion
title: Tick Marks | Thumb Track |  Ignite UI for Angular | Infragistics
```

File: en/components/slider/slider.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 _description: Ignite UI for Angular Slider control provides a rich input for value selection and delivers consistent experience across browsers.
```
Comment (zdrawku) [inline en/components/slider/slider.md]:
```suggestion
_description: Learn how to configure the Ignite UI for Angular slider tick marks, and improve your data visualization by using angular slider step with convenient thumb track
```

File: en/components/slider/slider.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
-_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Slider components, Angular Slider controls
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Slider components, Angular Slider controls, Slider ticks, Slider, ticks
```
Comment (zdrawku) [inline en/components/slider/slider.md]:
```suggestion
_keywords: tick marks, igniteui for angular, infragistics
```

File: en/components/slider/slider_ticks.md [hunk 8]
```diff
@@ -0,0 +1,250 @@
+---
+title: Slider Component
+_description: Ignite UI for Angular Slider control provides a rich input for value selection and delivers consistent experience across browsers.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Slider components, Angular Slider controls
+---
+
+## Slider Ticks
+**Slider ticks**, provide a new and more appealing way for data visualization, like a particular timeframe, days of the week and more. With this new functionality, the users are not obliged to interact with the slider in order to see what data range is being represented. It is extremely flexible, with regards to the control over positioning and orientation of the **ticks** and **tick labels**. The **ticks** can be turned **on/off**, as well as can be toggled between **primary**, **secondary** or **both**. In addition, this feature provides a way to  turn **on/of** **primary**, **secondary** **tick labels** or both. Tick labels can change their rotation form **horizontal** to **vertical** (**top to bottom** (90) or **bottom to top** (-90)).
```
Comment (zdrawku) [inline en/components/slider/slider_ticks.md]:
```suggestion
**Slider tick marks**, provide a new and more convenient way for data visualization, like a particular timeframe, days of the week and more. With this new functionality, the users are not obliged to interact with the Angular Slider in order to see what data range is being represented. It is extremely flexible, with regards to the control over positioning and orientation of the tick marks, labels and thumb track. The **ticks** can be turned **on/off**, as well as can be toggled between **primary**, **secondary** or **both**. In addition, this feature provides a way to  turn **on/of** **primary**, **secondary** **tick labels** or both. Tick labels can change their rotation form **horizontal** to **vertical** (**top to bottom** (90) or **bottom to top** (-90)).
```

File: en/components/slider/slider_ticks.md [hunk 9]
```diff
@@ -0,0 +1,250 @@
+---
+title: Slider Component
+_description: Ignite UI for Angular Slider control provides a rich input for value selection and delivers consistent experience across browsers.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Slider components, Angular Slider controls
+---
+
+## Slider Ticks
```
Comment (zdrawku) [inline en/components/slider/slider_ticks.md]:
```suggestion
## Slider Tick Marks
```

**Comment by Aleksandyr** [comment]
@zdrawku and @yoshimis there are some more changes that need to be applied over the slider topic as well, so we will hold the localization for a while.

**Comment by yoshimis** [comment]
@Aleksandyr  please ping me once it is ready for us to localize.

**Comment by zdrawku** [comment]
@yoshimis and @randriova the content is approved and ready to be localized

**Pull request merged**